DASCharter-2020.html

```diff
@@ -511,6 +498,24 @@ <h3 id="rec-track">
               "https://www.w3.org/2018/06/devices-sensors-wg-charter.html">https://www.w3.org/2018/06/devices-sensors-wg-charter.html</a>
             </p>
           </dd>
+        </dl>
+        <h3 id="tentative">
+          Tentative Deliverables
+        </h3>
+        <dl>
```

Comment:
We should probably define what is a "tentative deliverable", and in particular what are the requirements for such a tentative deliverable to become a normative specification.

```suggestion
        <p>
          To publish the <a href="https://www.w3.org/2020/Process-20200915/#fpwd">First
          Public Working Draft</a> of a document denoted as a tentative deliverable in
          this Charter, the Devices and Sensors Working Group must meet the applicable
          <a href="https://www.w3.org/2020/Process-20200915/#transition-reqs">
          requirements for advancement</a>.
        </p>
        <dl>
```

Reply:
From a process perspective, there is no difference between a deliverable and a tentative deliverables. Both are on the REC track, subject to Group decisions. It's up to the Working Group to decide how a tentative deliverable would be published as a WD on /TR. The difference here is in how we communicate: a tentative deliverable requires more thinking from the Group before getting published.

Reply:
@plehegar since that term is not established W3C charter language, do you think some sort of explanation along those lines is appropriate ("requires more thinking")? I'm mainly thinking of folks who have not been part of the AB/TAG Experiment and lack the context.

Reply:
How about something like:

**Tentative Deliverables**

Depending on the progress, the Group may also produce W3C Recommendations for the following documents:
 [...]

Reply:
SGTM.

Reply:
SGTM

Reply:
To be clear, I’m fine with this PR with or without the @plehegar’s clarification. I’ll leave it to @xfq and @plehegar to decide what exact language to include, if any.